src/openrc-init/openrc-init.c

```diff
@@ -27,16 +27,15 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
-#include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/reboot.h>
 #include <sys/wait.h>
+#include <utmp.h>
```

Comment:
this is dubious as we're using RUN_LVL which doesn't look POSIX, but the include is "right" for now